lib/TPP/Dialect/Tpp/TppUtils.cpp

```diff
@@ -272,6 +270,34 @@ bool isTppZero(linalg::GenericOp linalgOp, SmallVectorImpl<Value> *operands) {
   return true;
 }
 
+void splitFusedOp(tpp::FusedBrgemmOp fusedBrgemmOp, PatternRewriter &rewriter) {
```

Comment:
Can we just not fuse them (for now) in the first place?

Reply:
You can still have a stand-alone op, addressing the issue at the time of lowering ensures that `tpp.fused_brgemm` always works.
Anyway, I will need the split anyway to implement lowering to loops.